src/specs/components/site-header/demo/data--ec.js

```diff
@@ -28,6 +28,16 @@ module.exports = {
     code: 'en',
     overlay: {
       title: 'Select your language',
+      close: {
+        variant: 'ghost',
+        label: 'Close',
+        icon: {
+          path: '/icons.svg',
+          name: 'close',
+          size: 'm',
```

Comment:
the size should rely on user data? I mean, are they supposed to change it if they want, otherwise this would be probably better defined in the template

Reply:
unfortunately the size is different between EC and EU, so we cannot really put it in the twig template (and putting it in the story won't help).
I should add that in the doc